evm/rpc/main.py

```diff
@@ -0,0 +1,115 @@
+import json
+import logging
+
+from evm.exceptions import (
+    ValidationError,
+)
+
+from evm.rpc.modules import (
+    Eth,
+    EVM,
+)
+
+REQUIRED_REQUEST_KEYS = (
+    'id',
+    'jsonrpc',
+    'method',
+)
+
+
+def validate_request(request):
+    for key in REQUIRED_REQUEST_KEYS:
```

Comment:
*nitpick*

I have started gravitating to the following style because it captures all of the missing keys.

```python
missing_keys = set(REQUIRED_REQUEST_KEYS).difference(request.keys())
if missing_keys:
    raise ValueError("...")
```